src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form form-big"
+      action="#"
+      method="post"
+      data-qa="big"
+    >
```

Comment:
please pay attention to code formatting. read checklist
```suggestion
    <form 
      class="form form-big"
      action="#"
      method="post"
      data-qa="big"
    >
```

Reply:
and fix it everywhere